lighthouse-core/test/config/config-test.js

```diff
@@ -705,11 +706,24 @@ describe('Config', () => {
         plugins: ['lighthouse-plugin-simple'],
       };
       const config = new Config(configJson, {configPath: configFixturePath});
-      const groupIds = Object.keys(config.groups);
       assert.deepStrictEqual(config.audits.map(a => a.path),
         ['installable-manifest', 'metrics', 'redirects', 'user-timings']);
-      assert.ok(groupIds.length === 1);
-      assert.strictEqual(groupIds[groupIds.length - 1], 'lighthouse-plugin-simple-new-group');
+    });
+
+    it('should append and use plugin-prefixed groups', () => {
```

Comment:
@jburger424 I split the `should append audits and a group` in two because I was having trouble following it all in one :) Should be exactly the same functionality

Reply:
:+1: this is cleaner